doc/data/messages/u/unused-private-member/good.py

```diff
@@ -1 +1,9 @@
-# This is a placeholder for correct code for this message.
+class Fruit:
+    FRUITS = ["apple", "orange"]
+
+    def __print_color(self, name):
```

Comment:
Sorry for the piecemeal review. One last Q. Is there a reason you used two leading underscores rather than one? AFAIK double underscores indicate "name mangling," and if that's not relevant here, then I just didn't want to encourage it in "good.py". Thoughts?

Reply:
Since we're demonstrating examples for unused-private-member, it wouldn't make a lot of sense to do an example of "good" by not having a private member at all, since "bad" has one. So here I"m using it.

Reply:
I meant a private member with one underscore rather than two. But it's alright.

Reply:
Isn't one underscore protected, not private?

Reply:
Sorry I merged without waiting for the discussion to conclude. I think @clavedeluna is right, protected could be used in a child class, we do need a private member for this example.

Reply:
You're right, I didn't look at the source code for the check and wrongly assumed it was dealing with both flavors private and protected. Never mind!